test/integration/default/controls/service.rb

```diff
@@ -1,5 +1,14 @@
 # frozen_string_literal: true
 
+case platform[:family]
+when 'redhat', 'fedora'
+  config_plugin_file = '/etc/collectd.d/postgresql.conf'
+when 'debian', 'ubuntu'
+  config_plugin_file = '/etc/collectd/plugins/postgresql.conf'
+when 'suse'
+  config_plugin_file = '/usr/lib64/collectd/postgresql.conf'
+end
```

Comment:
Let's make this more Ruby-ish and use an `else`:

```suggestion
config_plugin_file = 
  case platform[:family]
  when 'redhat', 'fedora'
    '/etc/collectd.d/postgresql.conf'
  when 'suse'
    '/usr/lib64/collectd/postgresql.conf'
  else
    '/etc/collectd/plugins/postgresql.conf'
  end
```